### PR TITLE
Surface plots

### PR DESCRIPTION
**Pull request by ZviBaratz**
Hi there, 

If it's alright, I'll try to pick up the awesome work made here by @dangom - I created this new branch and merged from master so that everything is up to date. There were a couple of meaningless test failures I fixed but otherwise things seems to be in place.

I'm still familiarizing myself with the code and what exactly are all the changes that were made, trying to improve docs along the way, but if anyone has useful information about the state of things please do share. 

Thanks again to @dangom! Hope I can help get your code merged and widely used soon :)

### CODE REVIEW

**Comment by dangom** [comment]
Hi @ZviBaratz thanks for taking this up.

If I remember correctly, the issue with this PR was that the colorbars weren't respecting the `symmetric_cbar` argument. That's because in order to plot a single colorbar for the figure (instead of 1 for each plot) I had to hack together a `_colorbar_from_array` function, mimicking some of the functionality that I think was already written elsewhere in nilearn for other functions.

The code should work otherwise just fine.

**Comment by ZviBaratz** [comment]
@dangom Thank you so much! Good to know other than that bug everything should be alright. I will try to come up with something for the colorbar.

File: .gitignore [hunk 1]
```diff
@@ -34,10 +34,6 @@ tags
 *.tgz
 
 .idea/
+venv/
 
 doc/themes/nilearn/static/jquery.js
-!/doc/_build/
```
Comment (jeromedockes) [inline .gitignore:39]: why is this changed?

File: doc/themes/nilearn/static/nature.css_t [hunk 2]
```diff
@@ -1037,6 +1037,13 @@ table.field-list tr td.field-body p.first {
 table.field-list tr td.field-body p {
     margin-top: 0.5em;
 }
+
+/* Fix spacelessness between parameter name and description */
+dl.field-list span.classifier {
+    margin-left: 1ex;
+}
```
Comment (GaelVaroquaux) [inline doc/themes/nilearn/static/nature.css_t:1044]: Hum, it seems to me that this PR is mixing commits from your other PR and from the original one. It would be best to start from a clean PR, because it will be very hard to review.

Comment (ZviBaratz) [reply doc/themes/nilearn/static/nature.css_t:1044]: Indeed. I noticed it too late and tried to fix it but fell in a git-hole. I'll recreate the branch and open a new PR tomorrow.

**Comment by thomasbazeille** [comment]
@ZviBaratz can you site the original issue this is supposed to close and the PR you're starting from. It's a mess to follow otherwise! Thanks

**Comment by ZviBaratz** [comment]
This issue is closed and replaced with #2477.
